### DISCUSSION TITLE
Focusing on Laravel would make more progress.

### DISCUSSION THREAD

**Discussion by Wotan6891** [vitodeploy/vito#412] [Ideas]
Hello,

I congratulate you for this work.

It would be better to just focus on Laravel since you would get more benefit this way.

Laravel is growing more than other frameworks.

Greetings

**Comment by saeedvaziry** [comment]
Thanks for the feedback.

I am not looking to get benefits from this project at the moment.

My main focus is to build something to make developers' lives (including myself) easier. For now PHP and later others maybe.
